app.js

```diff
@@ -19,6 +19,10 @@ function loadEventListeners() {
   document.addEventListener('DOMContentLoaded', loadAllTasks);
   //Filter tasks 
   filter.addEventListener('keyup', filterTasks);
+  //Click favorite task
+  taskList.addEventListener('click', favoriteTask);
+  //Move favorite tasks
+  taskList.addEventListener('click', moveFavorites);
```

Comment:
Don't add two event listeners to one element. It should be one event type to one element. In case you want to run different logic depending what element was clicked make this logic inside the click event handler.